example_test.go

```diff
@@ -86,8 +87,11 @@ func ExampleParseWithClaims_customClaimsType() {
 	token, err := jwt.ParseWithClaims(tokenString, &MyCustomClaims{}, func(token *jwt.Token) (interface{}, error) {
 		return []byte("AllYourBase"), nil
 	})
+	if err != nil {
+		log.Fatal(err)
+	}
 
-	if claims, ok := token.Claims.(*MyCustomClaims); ok && token.Valid {
+	if claims, ok := token.Claims.(*MyCustomClaims); ok {
```

Comment:
Wouldn't it be "cleaner" to have something like this:

```go
if claims, ok := token.Claims.(*MyCustomClaims); ok && err == nil {
  fmt.Printf("%v %v", claims.Foo, claims.RegisteredClaims.Issuer)
} else {
  fmt.Println(err)
}
```

or probably more elaborate like

```go
if err != nil {
   fmt.Println(err)
} else if claims, ok := token.Claims.(*MyCustomClaims); ok {
   fmt.Printf("%v %v", claims.Foo, claims.RegisteredClaims.Issuer)
} else {
   fmt.Printf("ParseWithClaims returned an unknown claims type, cannot proceed")
}
```

Reply:
Absolutely. It didn't make much sense before printing out the error in the else clause as it would be nil. I updated to your latter example. How does it look?

Reply:
A regular `Println` would suffice, instead of `Printf`, but otherwise it looks good. Would you mind adjusting the other four occurences as well?

Reply:
Oops. Yep, replaced `Printf` with `Println` where I could.